app/config/override.yaml

```diff
@@ -0,0 +1,14 @@
+cert-exporter:
+  chart: "cert-exporter"
+# chart-operator must be installed first so the chart CRD is
+# created in the tenant cluster.
+chart-operator:
+  chart:     "chart-operator"
+  namespace: "giantswarm"
+# CoreDNS's Upgrade force is disabled to avoid affecting customer workloads.
+coredns:
+  useUpgradeForce: false
+net-exporter:
+  chart: "net-exporter"
+nginx-ingress-controller:
+  useUpgradeForce: false
```

Comment:
There is a comment about affecting customer workloads for coredns. Its also needed here.